core/drivers/clk/clk.c

```diff
@@ -37,8 +37,8 @@ struct clk *clk_alloc(const char *name, const struct clk_ops *ops,
 		panic();
 	}
 
-	if (name && ops->get_name) {
-		EMSG("Operator get_name and name are exclusive");
+	if (ops->get_name != clk_elt_name) {
```

Comment:
Why do we care?

Reply:
I though about consistency in argument  `name` of this function and what `clk_get_name()` will return. However, I agree it's not a real issue, I can remove this test.